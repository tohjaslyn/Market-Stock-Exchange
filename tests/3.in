ADD trader1 0
IMPORT trader1 NFL 10
ADD trader2 0  
TRADERS
SELL trader1 NFL 10 1
BUY trader2 NFL 10 2.5
EXPORT trader2 NFL 5 
BALANCE trader1
BALANCE trader2
TRADES
SAVE traders trades
BINARY traders.bin trades.bin
EXIT
