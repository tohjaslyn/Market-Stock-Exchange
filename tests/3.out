$ Success.
$ Trader now has 10.00 units of NFL.
$ Success.
$ trader1
trader2
$ No trades could be made, order added to sell book.
$ Product bought in entirety, trades as follows:
trader1->trader2: 10.00xNFL for $1.00.
$ Trader now has 5.00 units of NFL.
$ $10.00
$ $-10.00
$ trader1->trader2: 10.00xNFL for $1.00.
$ Success.
$ Success.
$ Have a nice day.
